src/scripts/main.js

```diff
@@ -1 +1,82 @@
 'use strict';
+
+const MOUSE_BUTTONS = {
+  0: 'left',
+  1: 'wheel',
+  2: 'right',
+};
+
+const body = document.querySelector('.root');
+
+const promiseOne = new Promise((resolve, reject) => {
+  body.addEventListener('mousedown', (clickEvent) => {
+    const { button } = clickEvent;
+
+    if (MOUSE_BUTTONS[button] === 'left'
+        || MOUSE_BUTTONS[button] === 'wheel'
+        || MOUSE_BUTTONS[button] === 'right'
+    ) {
+      resolve();
+    }
```

Comment:
> The firstPromise should be
resolved with a message First promise was resolved on a left click in the document